app/src/main/java/org/permanent/permanent/ui/public/PublicProfileEditAboutFragment.kt

```diff
@@ -0,0 +1,39 @@
+package org.permanent.permanent.ui.public
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.activity_main.*
+import org.permanent.permanent.R
+import org.permanent.permanent.databinding.FragmentEditAboutBinding
+import org.permanent.permanent.ui.PermanentBaseFragment
+import org.permanent.permanent.viewmodels.PublicProfileViewModel
+
+class PublicProfileEditAboutFragment: PermanentBaseFragment(){
+    private lateinit var viewModel: PublicProfileViewModel
+    private lateinit var binding: FragmentEditAboutBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = FragmentEditAboutBinding.inflate(inflater, container, false)
+        binding.executePendingBindings()
+        binding.lifecycleOwner = this
+        activity?.toolbar?.menu?.findItem(R.id.settingsItem)?.isVisible = false
```

Comment:
maybe we should move this login in the activity and have the decision in one place in the activity viewmodel